app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SessionsHelper
+  # Returns the current logged-in user (if any).
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
+    end
+  end
+  # Logs in the given user.
+  def log_in(user)
```

Comment:
```suggestion
  end

  # Logs in the given user.
  def log_in(user)
```
ditto for new line between methods. Please go through rest of this PR and apply the same